ci-operator/step-registry/upi/install/gcp/upi-install-gcp-commands.sh

```diff
@@ -434,11 +437,9 @@ if [ -f 02_lb_int.py ]; then # for workflow using internal load balancers
   # https://github.com/openshift/installer/pull/8582
   gcloud compute instance-groups unmanaged add-instances "${BOOTSTRAP_INSTANCE_GROUP}" "--zone=${ZONE_0}" "--instances=${INFRA_ID}-bootstrap"
 
-  if version_check "4.12"; then
```

Comment:
BTW, sounds like we also need update `version_check` a bit, just like what we did in https://github.com/openshift/release/pull/54286/files

Reply:
Ah, ignore it, it was already updated in upstream.

Reply:
oh, I just rebased from master.